packages/hooks/src/ContextProvider/types.ts

```diff
@@ -70,6 +70,9 @@ export interface ResultValues {
   rating?: number;
   price?: string | Array<string>;
   originalPrice?: string | Array<string>;
+  imageIds?: string[];
+  variantIds?: string[];
+  variantImageIds?: string[];
```

Comment:
I'm wondering why it's not just one field to hold a list of images? For examples:
```suggestion
 images?: string[];
```

Reply:
All 3 have different values and meaning, in order to get the right variant images please [see this](https://sajari.atlassian.net/wiki/spaces/SAJARI/pages/1209991336/Search+Result+Settings?focusedCommentId=1283096679#comment-1283096679)

Reply:
If so, I think we are opinionating Shopify settings here. Ideally, the component should know nothing regarding Shopify and render the images based on a single prop. We can compute the value of the prop via the [mapResultsField](https://github.com/sajari/sdk-react/blob/master/packages/search-ui/src/utils/mapResultFields.ts#L40), don't we?

Reply:
You are correct, I'll make the changes.